CHANGELOG.md

```diff
@@ -29,11 +28,14 @@ x.x.x
 - Add `lambda:ListTags` and `lambda:UntagResource` to `ParallelClusterUserRole` used by ParallelCluster API stack for cluster update.
 - Add `parallelcluster:cluster-name` tag to all resources created by ParallelCluster.
 - Do not allow setting `PlacementGroup/Id` when `PlacementGroup/Enabled` is explicitly set to `false`.
-- Restrict IPv6 access to IMDS to root and cluster admin users only.
+- Restrict IPv6 access to IMDS to root and cluster admin users only, when configuration parameter `HeadNode/Imds/Secured` is enabled.
 - Change the default root volume size from 35 GiB to the size of AMIs. The default can be overwritten in cluster configuration file.
+- Disable the compute fleet when the configuration parameter `Scheduling/SlurmQueues/ComputeResources/SpotPrice`
+  is lower than the minimum required Spot request fulfillment price.
 
 **BUG FIXES**
 - Fix default for disable validate and test components when building custom AMI. The default was to disable those components, but it wasn't effective.
+- Handle corner case when instance is just launched and the describe doesn't report yet all the EC2 info.
```

Comment:
let's highlight it's a corner case in the scaling logic (and not for example a corner case in a CLI copmmand).

Reply:
describe --> describe instance API ?